rust/src/editor.rs

```diff
@@ -29,6 +29,7 @@ use view::View;
 use tabs::update_tab;
 
 const MODIFIER_SHIFT: u64 = 2;
+const MODIFIER_COMMAND: u64 = 4;
```

Comment:
I want to move away from interpreting modifiers in the back-end (do_key is going away), so have been thinking of changing from testing MODIFIER_SHIFT to a bool, say `extend_selection`. I think it makes sense to have a separate `move_beginning_of_line`, as opposed to yet another option for `move_left`.